Extension/SourceEditorCommand.swift

```diff
@@ -4,24 +4,21 @@ import XcodeKit
 class SourceEditorCommand: NSObject, XCSourceEditorCommand {
 
     func perform(with invocation: XCSourceEditorCommandInvocation,
-                 completionHandler: @escaping (Error?) -> Swift.Void) {
+        completionHandler: @escaping (Error?) -> Swift.Void) {
 
         let uti = invocation.buffer.contentUTI
-        if uti != "com.apple.dt.playground" && uti != "public.swift-source" && uti != "com.apple.dt.playgroundpage"{
+        guard uti == "com.apple.dt.playground" || uti == "public.swift-source" || uti == "com.apple.dt.playgroundpage" else {
```

Comment:
There's a bug before.
Maybe multi `guard let` is too complicate and not best choice in this case.
Array contain maybe another choice.